pbs_light.go

```diff
@@ -362,18 +354,14 @@ func auction(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
 	if pbs_req.CacheMarkup == 1 {
 		cobjs := make([]*pbc.CacheObject, len(pbs_resp.Bids))
 		for i, bid := range pbs_resp.Bids {
-			bc := BidCache{
+			bc := &pbc.BidCache{
 				Adm:    bid.Adm,
 				NURL:   bid.NURL,
 				Width:  bid.Width,
 				Height: bid.Height,
 			}
-			buf := new(bytes.Buffer)
-			enc := json.NewEncoder(buf)
-			enc.SetEscapeHTML(false)
```

Comment:
This is a small/subtle difference from the current behavior, and I don't see a test for it... so just making sure it's ok.

`prebid_cache.go` transforms these objects into JSON with: `reqJSON, err := json.Marshal(pr)`

The impementation of json.Marshal is:

```
	e := &encodeState{}
	err := e.marshal(v, encOpts{escapeHTML: true})
```

The code you're deleting here has `enc.SetEscapeHTML(false)`.

Does this sound like trouble to you, or are you sure it's safe? I'm suspecting that most ad markup (`adm`) has some HTML in it.

Reply:
Thanks for catching this, I do think that we need SetEscapeHTML(false). I'll work on the implementation and add tests.